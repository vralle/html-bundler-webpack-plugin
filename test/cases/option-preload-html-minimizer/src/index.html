<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <!-- TEST: correctly minify the content value, keep the space between parameters -->
  <meta name="viewport" content="width=device-width initial-scale=1">
  <link href="./home.scss" rel="stylesheet">
  <script src="./main.js"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>