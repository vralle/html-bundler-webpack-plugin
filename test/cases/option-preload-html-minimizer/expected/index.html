<!doctype html><html><head><title>Home</title><meta name=viewport content="width=device-width initial-scale=1"><link rel="preload" href="js/main.bundle.js" as="script">
<link rel="preload" href="css/home.bundle.css" as="style">
<link rel="preload" href="fonts/open-sans-regular.woff2" as="font" type="font/woff2" crossorigin>
<link rel="preload" href="fonts/MaterialIcons-Regular.woff2" as="font" type="font/woff2" crossorigin>
<link href=css/home.bundle.css rel=stylesheet><script src=js/main.bundle.js></script></head><body><h1>Hello World!</h1></body></html>